src/container.ts

```diff
@@ -662,6 +710,47 @@ export enum MountPropagation {
   BIDIRECTIONAL = 'Bidirectional',
 }
 
+/**
+ * CPU and memory compute resources
+ */
+export interface IResources {
```

Comment:
We normally use `IXxx` naming for [imported](https://github.com/cdk8s-team/cdk8s-plus/blob/k8s-22/main/src/secret.ts#L24) resources, not for inputs. 

Just remove the `I` prefix.